android/modules/media/src/java/ti/modules/titanium/media/TiSound.java

```diff
@@ -426,6 +426,21 @@ public int getDuration()
 		return duration;
 	}
 
+	public boolean isMuted()
+	{
+		AudioManager audioManager =
+			(AudioManager) TiApplication.getInstance().getApplicationContext().getSystemService(Context.AUDIO_SERVICE);
```

Comment:
I did not guard it because other usages in that class do not as well (like `requestAudioFocus`), but that's not the best reason :-). Would love to know the nullability of these methods!